build-system/tasks/e2e/index.js

```diff
@@ -40,12 +40,13 @@ function installPackages_() {
 }
 
 function buildRuntime_() {
-  execOrDie('gulp build');
+  execOrDie('gulp clean');
```

Comment:
As mentioned in another comment, this is no longer necessary, since `dist()` already calls `clean()`.